integration-tests/tests/index.test.ts

```diff
@@ -23,11 +23,11 @@ const encryptConfig = JSON.stringify({
       },
     },
   },
-})
+}
 
 describe('encrypt and decrypt', async () => {
   test('can round-trip encrypt and decrypt', async () => {
-    const client = await newClient(encryptConfig)
+    const client = await newClient({ encryptConfig })
```

Comment:
Most of the changes on the Protect.js side will involve wrapping args in objects (instead of using positional args). The tests and types are the best reference for the new interface.